apps/condo/domains/notification/schema/_internalSendNotificationNewMobileAppVersionService.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * Generated by `createservice notification._internalSendNotificationNewMobileAppVersionService --type mutations`
+ */
+
+const get = require('lodash/get')
+const isEmpty = require('lodash/isEmpty')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT, INTERNAL_ERROR } } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema, getSchemaCtx } = require('@open-condo/keystone/schema')
+
+const { loadListByChunks } = require('@condo/domains/common/utils/serverSchema')
+const access = require('@condo/domains/notification/access/_internalSendNotificationNewMobileAppVersionService')
+const {
+    DEVICE_PLATFORM_IOS,
+    DEVICE_PLATFORM_ANDROID,
+    APP_MASTER_KEY,
+    APP_RESIDENT_KEY,
+    DEFAULT_TITLE_FOR_NEW_MOBILE_APP_VERSION_PUSH_NOTIFICATION,
+    DEFAULT_BODY_FOR_NEW_MOBILE_APP_VERSION_PUSH_NOTIFICATION,
+    MOBILE_APP_UPDATE_AVAILABLE_MESSAGE_PUSH_TYPE,
+} = require('@condo/domains/notification/constants/constants')
+const { MessageBatch, RemoteClient } = require('@condo/domains/notification/utils/serverSchema')
+const { Resident } = require('@condo/domains/resident/utils/serverSchema')
+const { RESIDENT, STAFF } = require('@condo/domains/user/constants/common')
+
+const _internalSendNotificationNewMobileAppVersionService = new GQLCustomSchema('_internalSendNotificationNewMobileAppVersionService', {
+    types: [
+        {
+            access: true,
+            type: `enum App { ${APP_RESIDENT_KEY}, ${APP_MASTER_KEY} }`,
```

Comment:
Too simple name. `App` can be related to miniapps / billing apps / mobiles apps. Maybe more clear name here?

Reply:
I think from the name of the mutation it’s clear that this is a mobile app